### PR TITLE
Updated Issues

### PR DESCRIPTION
**Pull request by Slothjam**
## v.1.21.1 - 4/04/2019
* Renamed src file for 'Undo': src/icons/0152_circleArrowLeft_a.svg
* Added 'Deprecating' column
	* Added 'Reload' icon back in.

### CODE REVIEW

File: src/CernerOneIcons-CSV.csv [hunk 1]
```diff
@@ -1,255 +1,255 @@
...
-x_symbol,src/icons/S_0011_x.svg,themeable ,
-circleLines_symbol,src/icons/S_0012_circleLines.svg,themeable ,
-catalogueNumber,src/icons/R_catalogueNumber.svg,themeable ,
-caution,src/icons/R_caution.svg,themeable ,
-consultInstructionsForUse,src/icons/R_consultInstructionsForUse.svg,themeable ,
-dateOfManufacture,src/icons/R_dateOfManufacture.svg,themeable ,
-manufacturer,src/icons/R_manufacturer.svg,themeable ,
\ No newline at end of file
+add,src/icons/0001_plus_a.svg,themeable ,,,Deprecating,
+search,src/icons/0002_magnifyingGlass_a.svg,themeable ,,,reload,src/icons/0152_circleArrow_a.svg
```
Comment (bjankord) [inline src/CernerOneIcons-CSV.csv]: Theres some odd formatting with this, I think you'll want to update the CSV to this: https://gist.githubusercontent.com/bjankord/67ecee693fae7ae29d6c5c650bbaecaa/raw/b6c31d9d821e768c1ee5b44434cae578bcffa6a8/icons.csv

Comment (Slothjam) [reply src/CernerOneIcons-CSV.csv]: Updated CSV for formatting

**Review by bjankord — APPROVED** [review]
LGTM

**Comment by cg5037** [comment]
+1

**Pull request merged**